### ISSUE TITLE
Use .pbf files instead .osm 

### ISSUE THREAD

**Issue by Uranus45** [ad-freiburg/pfaedle#10]
Protocol buffer files are much smaller than the .osm (XML) files. Also, pbf files can be parsed much faster then the XML files. There's open source project called [libosmium (https://github.com/osmcode/libosmium) which can be used for pbf parsing.

**Comment by patrickbr** [comment]
Indeed, this is something we have been planning for a while. We will give an update here as soon as we have implemented it :)

**Comment by kylerchin** [comment]
This issue can now be closed
